<?xml version="1.0"?>
<launch>
  <arg name="object_name" default="HV8" />
  <arg name="icpRefine" default="False" />
  <arg name="arch" default="PointNet_Pose" />
  <arg name="name" default="PointNet" />
  <arg name="resolution" default="50" />
  <node name="estimator_client" pkg="estimator" type="estimator_client.py" output="screen" >
    <param name="object_name" value="$(arg object_name)" />
    <param name="icpRefine" value="$(arg icpRefine)" />
    <param name="arch" value="$(arg arch)" />
    <param name="name" value="$(arg name)" />
    <param name="resolution" value="$(arg resolution)" />
  </node>

  <node name="estimator_server" pkg="estimator" type="estimator_server.py" output="screen" >
    <param name="object_name" value="$(arg object_name)" />
    <param name="name" value="$(arg name)" />
    <param name="arch" value="$(arg arch)" />
    <param name="resolution" value="$(arg resolution)" />
  </node>




  <arg name="gui" default="true"/>
  <param name="use_gui" value="$(arg gui)"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find pose_estimator_measure)/rviz/pose_estimator_measure_config.rviz" required="true" />
</launch>
